data/json/items/ammo.json

```diff
@@ -741,6 +741,24 @@
     "stack_size": 10,
     "fuel": { "energy": 1.6 }
   },
+  {
+    "type": "AMMO",
+    "id": "cac2powder",
+    "category": "spare_parts",
+    "price": 200,
+    "name": "calcium carbide premix",
```

Comment:
`name_plural` needed.

Reply:
is name_plural needed when the name itself is both singular and plural?

Reply:
If you don't provided it `"calcium carbide premixs"` will be used by default.  This exactly the case where it is needed.

Reply:
understood